monkey_island/cc/main.py

```diff
@@ -6,6 +6,11 @@
 import logging
 
 BASE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+PARENT_PATH = os.path.dirname(BASE_PATH)
```

Comment:
What is this doing in this PR?

Reply:
Yeah. I think this commit was before the common branch was even created 😅 
I can remove this, fix other CR comments, backmerge from develop and then from support common.
Then I'll merge after support common is merged.
You have any other suggestion?

Reply:
No, lets go with this